test/pylib/suite/python.py

```diff
@@ -213,7 +213,9 @@ async def run(self, options: argparse.Namespace) -> Test:
                     cc.execute(stmt)
                 cluster.prepare_cql_executed = True
             logger.info("Leasing Scylla cluster %s for test %s", cluster, self.uname)
-            self.args.insert(0, "--host={}".format(cluster.endpoint()))
+            self.args.insert(0, f"--host={cluster.endpoint()}")
```

Comment:
nit: Irrelevant "while at it" changes are not very welcome

Reply:
I made this change for consistency with the new `--scylla-log-filename` option formatting below. f-strings are preferable.